contribs/gmf/externs/gmfx.js

```diff
@@ -511,3 +514,36 @@ gmfx.TimePropertyResolutionEnum = {
  * }}
  */
 gmfx.TimeProperty;
+
+
+/**
+ * @typedef {{
+ *  data: Object,
```

Comment:
Not possible: `contribs/gmf/externs/gmfx.js:521: ERROR - Bad type annotation. type not recognized due to syntax error
ERR! compile  *  data: {{ short_url : string }},`

The best I can do is to replace `Object` by `Object.<string>`